app/database/queries/breeze_buddy/analytics.py

```
@@ -0,0 +1,317 @@
+                outcome_counts.count
+            ) as outcome_breakdown
+        FROM "{LEAD_CALL_TRACKER_TABLE}" lct
+        LEFT JOIN "{OUTBOUND_NUMBER_TABLE}" ou ON lct.outbound_number_id = ou.id
+        LEFT JOIN LATERAL (
+            SELECT lct2.outcome, COUNT(*) as count
+            FROM "{LEAD_CALL_TRACKER_TABLE}" lct2
+            {where_clause.replace('lct.', 'lct2.')}
+            GROUP BY lct2.outcome
+        ) outcome_counts ON true
+        {where_clause};
+    """
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)